ui/component/viewers/videoViewer/internal/videojs-events.jsx

```diff
@@ -223,6 +273,8 @@ const VideoJsEvents = ({
     player.on('volumechange', resolveCtrlText);
     player.on('volumechange', onVolumeChange);
     player.on('error', onError);
+    // first play tracking, used for initializing the watchman api
+    player.on('tracking:firstplay', doTrackingFirstPlay);
```

Comment:
Nitpick :)  --  this comment is probably unnecessary.
- `tracking:firstplay` is already self-documenting, so no need to repeat the obvious.
- It does more than initializing watchman, so it's misleading at first glance. There is already documentation where the `doTrackingFirstPlay` is defined, no need to repeat at the usages.